### PR TITLE
Arabic is applied to ( textpresets ,Localizable ,infoPlist ) files

### PR DESCRIPTION
**Pull request by Zeglaty**
I made the app support the Arabic language , but my only problem is that I do not know how to edit you keyboard to make it provide Arabic letters . 
so I think its your tern now.

### CODE REVIEW

File: Vocable/Supporting Files/ar.lproj/textpresets.json [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        "أنا أريد زوار",
+        "أرجوك كن هادءً",
+        "أريد أن أتكلم",
+        "أرجوك إفتح جهاز التلفاز",
+        "أرجوك أغلق جهاز التلفاز",
+        "أرجوك إرفع مستوى الصوت",
+        "أرجوك إخفظ مستوى الصوت",
+        "أرجوك إفتح الستائر",
+        "أرجوك أغلق الستائر",
+        "أرجوك إفتك النافذه",
```
Comment (ameirwillowtree) [inline Vocable/Supporting Files/ar.lproj/textpresets.json:79]: @Zeglaty did you mean to say:
أرجوك افتح النافذه

File: Vocable/Supporting Files/ar.lproj/textpresets.json [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        "كيف كانت عطلة نهاية الأسبوع؟",
+        "وداعاً",
+        "حسناً",
+        "سيئ",
+        "جيد",
+        "هذا منطقي",
+        "أعجبتني",
+        "أرجوك توقف",
+        "أنا لا أتفق",
+        "أرجوك أعد ماقلته"
```
Comment (ameirwillowtree) [inline Vocable/Supporting Files/ar.lproj/textpresets.json:62]: this should probably be
أرجوك أعد ماقلت

File: Vocable/Supporting Files/ar.lproj/textpresets.json (hunk 1 above)
Comment (Zeglaty) [reply Vocable/Supporting Files/ar.lproj/textpresets.json:79]: yes

File: Vocable/Supporting Files/ar.lproj/textpresets.json (hunk 2 above)
Comment (Zeglaty) [reply Vocable/Supporting Files/ar.lproj/textpresets.json:62]: i don't think so

Comment (Zeglaty) [reply Vocable/Supporting Files/ar.lproj/textpresets.json:62]: الهاء في الأخر تعود علي المتكلم

Comment (ameirwillowtree) [reply Vocable/Supporting Files/ar.lproj/textpresets.json:62]: that makes sense

**Review by Clstroud — APPROVED** [review]
Thanks for contributing these translations @Zeglaty! We are going to merge this in and do some testing to ensure we are properly handling things like Right-to-Left UI and localized date/number formatting. I can't guarantee this will ship in the next App Store release, but we will do our best to include it as soon as possible. Features like Arabic keyboard support will take some additional time to design and develop.

**Pull request merged**